src/guard/is-try/is-try.ts

```diff
@@ -0,0 +1,5 @@
+import type { Try } from '../../type/try'
+
+export function isTry<T extends Try<unknown>>(x: T | unknown): x is T {
+    return x !== null && x !== undefined && typeof x === 'object' && ('success' in x || 'failure' in x)
```

Comment:
```suggestion
    return true
```
everything is a try in the new definition I guess :laughing: